### DISCUSSION TITLE
[Lazer MP] Map parameters

### DISCUSSION THREAD

**Discussion by Veldora2k** [ppy/osu#16008] [Ideas] [answered]
Hi, i would like to see basic beatmap parameters like i can see star difficulty in this window :) 
Screenshot [https://i.imgur.com/gO6IFEL.png](https://i.imgur.com/gO6IFEL.png)

Thanks for awesome lazer :)

**Comment by bdach** [comment]
you can click on the difficulty name to open the beatmap overlay which shows these values, so not sure this needs to be in the tooltip.

**Reply by Veldora2k** [reply]
Yes, i know. We can also see star rating in the beatmap overlay so why it shows in this small gray window? Because you don't need to open window ¯\_( ͡❛ ͜ʖ ͡❛)_/¯

**Reply by smoogipoo** [reply]
I do agree with having this info available on hover. AR, CS, and BPM specifically are very nice to know, to the point where having to open another overlay all the time would get unnecessarily annoying.

**Comment by Theighlin** [comment, marked as answer]
#11306
